### PR TITLE
Update homepage

### LINKED ISSUE #8171: Amend the ubuntu.com homepage
Amends have come in from the PdMs and Mark on the homepage designs. They can be found in the [copy doc](https://docs.google.com/document/d/1ySJxQbqVdeH4Tra0zwBm2Tn0s56kFGnEF7d8xDRTxwU/edit).

@lyubomir-popov Has made a template a template which could help and be found here: https://drive.google.com/drive/folders/1cmolEmQf11XwdiKH1YOGenGvuM1BD9uf

### PR DESCRIPTION
**Pull request by pmahnke**
## Done
- Complete rebuilt the homepage content.
_Copy and layout has been approved by the PMs_

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1ySJxQbqVdeH4Tra0zwBm2Tn0s56kFGnEF7d8xDRTxwU/edit?ts=5f450201#heading=h.vmiyl9gud145)

## Issue / Card

Fixes #8171

### CODE REVIEW

**Comment by pmahnke** [comment]
@anthonydillon and @meralonso this is now up to date (pending more text changes) and I attempted to do the top grey logo section.  I could not make it work with all the logos on the right, so I tried this.  Happy for more ideas.

**Comment by caldav** [comment]
Can we get multiple modal forms on the page? Eg. the private cloud contact link loads the fallback OpenStack form as opposed to the modal one.

**Comment by anthonydillon** [comment]
> Can we get multiple modal forms on the page? Eg. the private cloud contact link loads the fallback OpenStack form as opposed to the modal one.

No, we can not currently. But I was assured that was ok.

**Comment by anthonydillon** [comment]
@pmahnke @meralonso what do you think of this?

**Comment by meralonso** [comment]
@anthonydillon could you please follow this layout?

Homepage v4 Copy

**Resized logos below:**

Modern enterprise open source
[AT&T](https://assets.ubuntu.com/v1/d8f890fb-logo-at%26t.svg)
[Google Cloud](https://assets.ubuntu.com/v1/901d5a91-Google+Cloud.png)
[Netflix](https://assets.ubuntu.com/v1/9dccb64d-Neflix.png)
[AWS](https://assets.ubuntu.com/v1/43d45da9-Anbox-logo-aws.png)
[Azure](https://assets.ubuntu.com/v1/d5e90eed-Microsoft+Azure.png)

[Business Insider](https://assets.ubuntu.com/v1/c79bf79d-Business+Insider.png)
[Tech Crunch](https://assets.ubuntu.com/v1/b1610fd0-TechCrunch.png)
[BBC](https://assets.ubuntu.com/v1/5186724f-BBC.png)
[WSJ](https://assets.ubuntu.com/v1/5311e1ca-WSJ.png)
[Forbes](https://assets.ubuntu.com/v1/3dd576dc-Forbes.png)

Open source security:
[Nist](https://assets.ubuntu.com/v1/4f916821-NIST_logo.png)
[CSEC](https://assets.ubuntu.com/v1/906a6abb-csec_logo.png)
[@sec](https://assets.ubuntu.com/v1/acd3f22d-%40sec_logo.png) 
[CIS](https://assets.ubuntu.com/v1/f84dc60c-cis_logo.png)
[DISA](https://assets.ubuntu.com/v1/8049bb5f-DISA_logo.png)

**Comment by meralonso** [comment]
@anthonydillon I have updated layout - shall I export customer logos as one image or maybe crop the logos so it fits a bit better?

Homepage v5

**Comment by lyubomir-popov** [comment]
@anthonydillon https://pastebin.canonical.com/p/MPkcTWVzHJ/

File: static/sass/_pattern_takeovers.scss [hunk 1]
```diff
@@ -3,13 +3,15 @@
 @mixin p-takeovers($name) {
   .p-takeover--#{$name} {
     @extend %vf-strip;
+
     background-blend-mode: multiply, multiply, normal, normal;
-    @content;
-    background-position: top right, top left, right bottom -1px, left top;
+
+    @content background-position: top right, top left, right bottom -1px, left top;
```
Comment (barrymcgee) [inline static/sass/_pattern_takeovers.scss:9]: `@content` needs to be on it's own line with a semi-colon, does it not?

File: static/sass/styles.scss [hunk 2]
```diff
@@ -772,3 +774,10 @@ body {
   right: 0;
   top: 0;
 }
+
+.article-title {
+  -webkit-box-orient: vertical;
+  display: -webkit-box;
```
Comment (barrymcgee) [inline static/sass/styles.scss:780]: I've never seen this display property before nor can I find anything on it by Googling.. what does it do? Does autoprefixer polyfill not webkit browsers?

File: templates/base_index.html [hunk 3]
```diff
@@ -1,330 +1,1583 @@
-
 {% extends "templates/base.html" %}
 
-{% block title %}The leading operating system for PCs, IoT devices, servers and the cloud{% endblock %}
+{% block title %}Enterprise Open Source and Linux{% endblock %}
+{% block meta_description %}Ubuntu is modern open source and Linux for the enterprise server, desktop, cloud, and IoT.{% endblock meta_description %}
```
Comment (barrymcgee) [inline templates/base_index.html:4]:
```suggestion
{% block meta_description %}Ubuntu is the modern, open source operating system on Linux for the enterprise server, desktop, cloud, and IoT.{% endblock meta_description %}
```

File: templates/base_index.html [hunk 4]
```diff
@@ -1,330 +1,1583 @@
...
+          loading="lazy",
+          attrs={"class": "p-inline-images__logo"},
+          ) | safe
+        }}
+      </li>
+      <li class="p-inline-images__item">
+        {{
+          image(
+          url="https://assets.ubuntu.com/v1/156ddbf8-rexroth_logo.svg",
+          alt="rexroth logo",
```
Comment (barrymcgee) [inline templates/base_index.html:920]:
```suggestion
          alt="Rexroth logo",
```

File: templates/base_index.html [hunk 5]
```diff
@@ -1,330 +1,1583 @@
...
+          loading="lazy",
+          attrs={"class": "p-inline-images__logo"},
+          ) | safe
+        }}
+      </li>
+      <li class="p-inline-images__item">
+        {{
+          image(
+          url="https://assets.ubuntu.com/v1/0e4d0d27-arm_logo.svg",
+          alt="arm logo",
```
Comment (barrymcgee) [inline templates/base_index.html:933]:
```suggestion
          alt="ARM logo",
```

File: templates/base_index.html [hunk 6]
```diff
@@ -1,330 +1,1583 @@
...
+          loading="lazy",
+          attrs={"class": "p-inline-images__logo"},
+          ) | safe
+        }}
+      </li>
+      <li class="p-inline-images__item">
+        {{
+          image(
+          url="https://assets.ubuntu.com/v1/f5dc1d41-pal-logo.svg",
+          alt="pal logo",
```
Comment (barrymcgee) [inline templates/base_index.html:1321]:
```suggestion
          alt="PAL logo",
```

File: templates/base_index.html [hunk 7]
```diff
@@ -1,330 +1,1583 @@
...
+
+<script>
+  function getPrimaryParentLanguage() {
+    /**
+    * Get the primary parent language
+    *
+    * Get the user's default language, and if it's a sub-language,
+    * infer the parent language - e.g. for "en-GB", infer "en"
+    */
+    var language = navigator.language || navigator.userLanguage || navigator.browserLanguage || navigator.systemLanguage;
```
Comment (barrymcgee) [inline templates/base_index.html:1497]:
```suggestion
    let language = navigator.language || navigator.userLanguage || navigator.browserLanguage || navigator.systemLanguage;
```

File: templates/base_index.html [hunk 8]
```diff
@@ -1,330 +1,1583 @@
...
+
+    if (language.indexOf('-') !== -1 ) {
+      language = language.split('-')[0];
+    }
+
+    return language
+  };
+
+  // get the users language and remove any extra detail suffix (e.g. -gb)
+  var language = getPrimaryParentLanguage();
```
Comment (barrymcgee) [inline templates/base_index.html:1507]:
```suggestion
  const primaryParentLanguage = getPrimaryParentLanguage();
```

File: templates/base_index.html [hunk 9]
```diff
@@ -1,330 +1,1583 @@
...
+
+  // get notices matching the user language
+  var notices = document.querySelectorAll(".notice[lang=" + language + "]");
```
Comment (barrymcgee) [inline templates/base_index.html:1510]:
```suggestion
  const notices = document.querySelectorAll(".notice[lang=" + primaryLanguage + "]");
```

**Review by barrymcgee — CHANGES_REQUESTED** [review]
Some small comments inline

File: static/sass/styles.scss (hunk 2 above)
Comment (anthonydillon) [reply static/sass/styles.scss:780]: It a webkit only property which allows us to use the new line-clamp (-webkit-line-clamp). Which is the only CSS only way to truncate lines over more than one line. https://css-tricks.com/line-clampin/

File: templates/base_index.html (hunk 3 above)
Comment (anthonydillon) [reply templates/base_index.html:4]: Thanks :+1:

File: templates/base_index.html (hunk 7 above)
Comment (anthonydillon) [reply templates/base_index.html:1497]: Let doesn't work in older browers and this is not compiled.

Comment (barrymcgee) [reply templates/base_index.html:1497]: We supporting IE10? https://caniuse.com/#feat=let

File: templates/base_index.html (hunk 9 above)
Comment (anthonydillon) [reply templates/base_index.html]: Done

File: templates/base_index.html (hunk 8 above)
Comment (anthonydillon) [reply templates/base_index.html]: Done.

File: templates/base_index.html (hunk 6 above)
Comment (anthonydillon) [reply templates/base_index.html]: Done

File: templates/base_index.html (hunk 5 above)
Comment (anthonydillon) [reply templates/base_index.html]: Done

File: templates/base_index.html (hunk 4 above)
Comment (anthonydillon) [reply templates/base_index.html]: Done

File: static/sass/_pattern_takeovers.scss (hunk 1 above)
Comment (anthonydillon) [reply static/sass/_pattern_takeovers.scss]: Done

**Comment by anthonydillon** [comment]
@barrymcgee all done, ready for review again. Can you update the labels please.

**Review by barrymcgee — APPROVED** [review]

**Review by caldav — APPROVED** [review]
:+1:

**Pull request merged**